package.json

```diff
@@ -1,97 +1,120 @@
-    "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --runInBand --coverage",
-    "audit-dep": "npm audit --json | node_modules/.bin/audit-filter --nsp-config=.nsprc --audit=-",
-    "package": "mkdir ${ZIP_NAME} && cp package.json package-lock.json ${ZIP_NAME}/ && cp -r .build/src/* ${ZIP_NAME}/ && cd ${ZIP_NAME} && npm ci --production && rm package.json package-lock.json && zip -qr ../${ZIP_NAME}.zip .",
-    "security-checks": "npm audit --audit-level=critical && git secrets --register-aws && git secrets --scan && git log -p | scanrepo",
-    "tools-setup": "echo 'nothing to do'"
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm-run-all audit lint prettier",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-push": "npm-run-all security-checks build test"
+    }
   },
-  "author": "",
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-typescript": "^7.6.0",
-    "@types/jest": "^24.0.19",
-    "@types/jest-plugin-context": "^2.9.2",
-    "@types/joi": "^14.3.3",
-    "@types/lambda-tester": "^3.5.1",
-    "@types/lodash": "^4.14.142",
-    "@types/node": "^12.11.2",
-    "@types/request": "^2.48.1",
-    "@types/supertest": "^2.0.8",
-    "audit-filter": "^0.5.0",
-    "aws-lambda-mock-context": "^3.2.1",
-    "babel-jest": "^24.9.0",
-    "gulp": "^4.0.2",
-    "husky": "^3.0.8",
-    "jest": "^24.9.0",
-    "jest-cucumber": "^2.0.11",
-    "jest-plugin-context": "^2.9.0",
-    "jest-sonar-reporter": "^2.0.0",
-    "lambda-tester": "^3.6.0",
-    "lodash": "^4.17.19",
-    "serverless": "^2.24.0",
-    "serverless-dynamodb-local": "^0.2.38",
-    "serverless-offline": "5.12.0",
-    "serverless-plugin-tracing": "^2.0.0",
-    "serverless-plugin-typescript": "^1.1.9",
-    "standard": "^14.3.1",
-    "supertest": "4.0.2",
-    "ts-jest": "^26.1.1",
-    "ts-node": "^8.4.1",
-    "tslint": "^5.20.0",
-    "tslint-jsdoc-rules": "^0.2.0",
-    "typescript": "3.6.4"
+  "scripts": {
+    "start": "cross-env API_VERSION=${npm_package_version} NODE_ENV=local BUCKET=develop serverless offline start",
+    "dev": "cross-env SLS_DEBUG=* VERSION=${npm_package_version} concurrently \"npm start\" \"npm run test:unit:watch\"",
+    "test:unit": "cross-env NODE_ENV=test API_VERSION=${npm_package_version} jest",
+    "test:unit:watch": "npm run test:unit -- --watch",
+    "test:unit:coverage": "cross-env COVERAGE=true npm run test:unit -- --coverage",
+    "test-i": "npm run test:integration",
+    "test:integration": "cross-env NODE_ENV=test echo 'integration tests to be added'",
+    "test:integration:watch": "echo 'integration tests --watch to be added'",
+    "test:integration:coverage": "cross-env COVERAGE=true echo 'integration tests coverage to be added'",
+    "test": "npm-run-all lint test:unit:coverage test:integration:coverage",
```

Comment:
If uses in the pipeline I think we can change this script to only run the unit test coverage since there is a script in the pipeline that already runs the integration test for the service (`test-i`) otherwise we will end up running the integration test twice in the pipeline